app/code/Magento/Sales/Controller/Adminhtml/Order/Invoice/Save.php

```diff
@@ -166,10 +166,6 @@ public function execute()
 
             $invoice = $this->invoiceService->prepareInvoice($order, $invoiceItems);
 
-            if (!$invoice) {
```

Comment:
Won't this cause an issue now with the testExecuteNotValidPost() test? - need to see